testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/SFSBHibernate2LcacheStats.java

```diff
@@ -79,6 +79,7 @@ public void setupConfig() {
             Configuration configuration = new Configuration().setProperty(AvailableSettings.USE_NEW_ID_GENERATOR_MAPPINGS,
                     "true");
             configuration.getProperties().put(AvailableSettings.JTA_PLATFORM, JBossAppServerJtaPlatform.class);
+            configuration.getProperties().put(AvailableSettings.TRANSACTION_COORDINATOR_STRATEGY, "jta");
```

Comment:
ORM 5.0.x added the TRANSACTION_COORDINATOR_STRATEGY="jta" setting requirement but it wasn't until later in ORM 5.1, that a bug fix (HHH-11024) required this to be set, that impacts schema drop code.

> git annotate ./hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java | grep hibernate.transaction.coordinator_class
> 7990630a	(Steve Ebersole	2015-07-27 17:05:41 -0500	184)	String TRANSACTION_COORDINATOR_STRATEGY = "hibernate.transaction.coordinator_class";

The summary is that a Hibernate ORM bug was fixed in 5.1, that depends on the application correctly specifying TRANSACTION_COORDINATOR_STRATEGY="jta".